skyvern/config.py

```diff
@@ -61,14 +61,15 @@ class Settings(BaseSettings):
     # LLM Configuration #
     #####################
     # ACTIVE LLM PROVIDER
-    LLM_KEY: str = "OPENAI_GPT4V"
+    LLM_KEY: str = ""
     # COMMON
     LLM_CONFIG_MAX_TOKENS: int = 4096
     LLM_CONFIG_TEMPERATURE: float = 0
     # LLM PROVIDER SPECIFIC
-    ENABLE_OPENAI: bool = True
+    ENABLE_OPENAI: bool = False
     ENABLE_ANTHROPIC: bool = False
     ENABLE_AZURE: bool = False
+    ENABLE_GEMINI: bool = True
```

Comment:
We shouldn't enable it by default. If you want to enable it on your local, add a row to your local `.env` file:
```
ENABLE_GEMINI=true
```

```suggestion
    ENABLE_GEMINI: bool = False
```